client/app/components/CreateEventModal.jsx

```diff
@@ -0,0 +1,181 @@
+      <option value={room.id} key={room.id}>{room.title}</option>
+    ));
+
+    return (
+      <Modal
+        show={this.props.showModal}
+        onHide={this.props.closeModal}>
+
+        <Modal.Header closeButton>
+          <Modal.Title>Create event</Modal.Title>
+        </Modal.Header>
+
+        <Modal.Body>
+          {this.state.showErrorMessage ?
+            <Alert bsStyle="danger">
+              Error occured while trying to save event.
+            </Alert>
+            : null}
+
+          <form>
+            <FormGroup>
+              <ControlLabel>Name:</ControlLabel>
+              <FormControl
+                type="text"
+                value={this.state.summary}
+                onChange={this._handleTextFieldChange}
+                name="summary" />
+            </FormGroup>
+            <FormGroup>
+              <ControlLabel>Description:</ControlLabel>
+              <FormControl
+                type="text"
+                value={this.state.description}
+                onChange={this._handleTextFieldChange}
+                name="description" />
+            </FormGroup>
+            <FormGroup>
+              <ControlLabel>Start time:</ControlLabel>
+              <DateTimeField
+                dateTime={this.state.startTime}
+                format={DATE_FORMAT}
+                inputFormat={DATE_FORMAT}
+                onChange={this._handleStartTimeChange} />
+            </FormGroup>
+            <FormGroup>
+              <ControlLabel>End time:</ControlLabel>
+              <DateTimeField
+                dateTime={this.state.endTime}
+                format={DATE_FORMAT}
+                inputFormat={DATE_FORMAT}
+                onChange={this._handleEndTimeChange} />
+            </FormGroup>
+            <FormGroup>
+              <ControlLabel>Location:</ControlLabel>
+              <FormControl
+                componentClass="select"
+                onChange={this._handleLocationChange}
+                name="location">
+                {conferenceRoomsOptions}
+              </FormControl>
+            </FormGroup>
+          </form>
+        </Modal.Body>
+
+        <Modal.Footer>
+          <Button onClick={this.props.closeModal}>Close</Button>
+          <Button
+            bsStyle="primary"
+            onClick={this._saveChanges}>
+            Save changes
+          </Button>
+        </Modal.Footer>
+      </Modal>
+    );
+  }
+
+
+  _handleTextFieldChange(e) {
+    const value = e.target.value;
+    const name = e.target.name;
+
+    this.setState({ [name]: value });
+  }
+
+  _handleLocationChange(e) {
+    const conferenceRoomId = e.target.value;
+
+    // only temporary, waiting for event API improvement
+    const conferenceRoomName = this.props.conferenceRooms.find(room =>
+      room.id === parseInt(conferenceRoomId, 10)
+    ).title;
+
+    this.setState({
+      location: conferenceRoomName,
+      conferenceRoomId
```

Comment:
Convention is to put the shortcut initializers (using the ES6 notation) at the top and `key: value` initializers after that